amd_openvx_extensions/amd_rpp/source/tensor/LensCorrection.cpp

```diff
@@ -202,8 +202,8 @@ static vx_status VX_CALLBACK uninitializeLensCorrection(vx_node node, const vx_r
     delete data->pDstDesc;
     delete data->pTableDesc;
     if (data->deviceType == AGO_TARGET_AFFINITY_CPU) {
-        delete[] data->pRowRemapTable;
-        delete[] data->pColRemapTable;
+        free(data->pRowRemapTable);
```

Comment:
Why are we using free?

Reply:
For these two variables, we are using calloc. So we shud dealloc them using free